docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -43,6 +43,7 @@ supported databases are MySQL, PostgreSQL, and Oracle.
     resource-groups.config-db-url=jdbc:mysql://localhost:3306/resource_groups
     resource-groups.config-db-user=username
     resource-groups.config-db-password=password
+    resource-groups.max-interval=2
```

Comment:
No need to add the config property here. Please update `Database resource group manager properties` instead.